content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods/at_confirmation.rst

```diff
@@ -0,0 +1,79 @@
+===========================
+At confirmation reservation
+===========================
+
+The *At Confirmation* reservation method reserves products **only** when a sales order (SO) is
+confirmed, **and** if enough stock of the products included in the :abbr:`SO (sales order)` is
+already available.
+
+.. seealso::
+   :doc:`About reservation methods <../reservation_methods>`
+
+Configuration
+=============
+
+To set the reservation method to *At Confirmation*, navigate to :menuselection:`Inventory app -->
+Configuration --> Operations Types`.
+
+In the :guilabel:`General` tab, locate the :guilabel:`Reservation Method` field, and select
+:guilabel:`At Confirmation`.
+
+.. image:: at_confirmation/at-confirmation-operations-type.png
+   :align: center
+   :alt: Reservation method field on delivery order operation type form.
+
+Workflow
+========
+
+To see the *At Confirmation* reservation method in action, create a new :abbr:`SO (sales order)` by
+navigating to :menuselection:`Sales app --> New`.
+
+Add a customer in the :guilabel:`Customer` field, then, in the :guilabel:`Order Lines` tab, click
+:guilabel:`Add a product`, and select a product to add to the quotation from the drop-down menu.
+Finally, in the :guilabel:`Quantity` column, adjust the desired quantity of the product to sell.
+
+Once ready, click :guilabel:`Confirm` to confirm the sales order.
+
+Click the green :guilabel:`📈 (forecast graph)` icon on the product line to reveal the product's
+:guilabel:`Availability` tooltip. This tooltip reveals the reserved number of units for this order,
+which, in this case, is `100 Units`.
+
+.. image:: at_confirmation/at-confirmation-availability-tooltip.png
+   :align: center
+   :alt: Confirmed sales order with product availability tooltip selected.
+
+.. admonition:: Forecasted Report
+
+   To see all the factors that affect product reservation, click the :guilabel:`View Forecast`
+   internal link arrow to view the :guilabel:`Forecasted Report` page.
```

Comment:
Let me look into that one -- as far as I understood, I could always see the link despite availability. I'll do some testing and update accordingly.

Reply:
So, I did some testing and found I am able to see the link with 0 product in stock, and even negative forecasted stock. However, I did find that the availability tooltip changes based on the quantity on hand. So I added this admonition below:

.. note::
   If there is **not** sufficient quantity of stock for the product included in the |SO|, the
   :guilabel:`📈 (forecast graph)` icon is red, instead of green.

   Instead of revealing the reserved number of units for the order, the :guilabel:`Availability`
   tooltip reads :guilabel:`Available`, and reveals the available number of units (e.g., `0 Units`).

Thanks for pointing this out! Suggestion implemented ✅